app/ui/browser-modern/views/browser/tabbar/tab.jsx

```diff
@@ -103,7 +106,11 @@ class Tab extends Component {
   }
 
   render() {
-    return (
+    const {
+      connectDropTarget = _ => _,
+      connectDragSource = _ => _,
```

Comment:
I had a mild stroke looking at this. Default props might be better in this case, but we sure love futuristic js so I have no opinions I guess.